cordapp-example/clients/src/main/kotlin/com/example/server/Server.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.server
+
+import net.corda.client.jackson.JacksonSupport
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.Banner
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.context.annotation.Bean
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter
+
+/**
+ * A Spring Boot application.
+ */
+
+@SpringBootApplication
+open class Server {
+
+    /**
+     * Spring Bean that binds a Corda Jackson object-mapper to HTTP message types used in Spring.
+     */
+    @Bean
+    open fun mappingJackson2HttpMessageConverter(@Autowired rpcConnection: NodeRPCConnection): MappingJackson2HttpMessageConverter {
```

Comment:
@EricMcEvoyR3 What's this used for?

Reply:
It's needed for JSON serialisation. Without it Spring translate the state data into a different format. It's because Spring is a separate implementation of this JAX-RS standard to Jetty and thus deals with JSON differently. Without it the, there is trouble displaying the JSON on the front-end.